wca-guidelines.md

```diff
@@ -47,6 +47,7 @@ To be more informative, each Guideline is classified using one of the following
 
 - 2c+) [ADDITION] First-time competitors should register using their legal name. They may register using a reasonable nickname, at the discretion of the WCA Delegate.
 - 2c++) [ADDITION] Competitors must not provide intentionally misleading information, and returning competitors should provide information consistent with past information (e.g. exact name and WCA ID).
+- 2c1+) [EXAMPLE] For example, the organization team may require a confirmation e-mail to be answered before a particular date, pre-payment of the registration fee, or paying due registration fees of past competitions. The organization team should carefully consider the fairness of the requirements.
```

Comment:
Should we mention that these requirements should be clearly stated on the competition's website?
It's fairly obvious to us, but we wouldn't want some organizers changing the requirements without the competitors knowing!

Reply:
The organizers can't change the information tab of the WCA website once the competition is confirmed, so maybe that's the place where they should clearly state those requirements?

Reply:
Yes, a good idea to clarify that such requirements must be clearly stated on the comp website.

Reply:
I initially meant to reply here but obviously messed it up, so please see my reply on the main discussion :p